### ISSUE TITLE
[Bug]: bing.com search not working in Floorp

### ISSUE THREAD

**Issue by decryptedchaos** [Floorp-Projects/Floorp#1320]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [ ] I have included relevant screenshots or console outputs.
- [X] I have checked that this problem is not occurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Simply put,  Floorp will not work with bing.com search  i believe its related to the fact that bing.com appears to open a new tab for results and Floorp is blocking that newtab  

### Reproduction Steps

Search something from bing.com 

Nothing happens

### Expected Behavior

Allow the popup tab i guess?  adding it to an exception does NOT work so its something deeper. 

### Actual Behavior

See summary

### Screenshots

pointless??  

### Browser Version

11.16.0 (64-bit)

### Operating System

Linux

**Comment by regularentropy** [comment]
![image](https://github.com/user-attachments/assets/8ae699fc-b35b-4171-adb6-87108865e598)
Cannot confirm the issue.

**Comment by creeper-0910** [comment]
Floorp may block pop-up displays, but it does not block new tabs from opening.

**Comment by decryptedchaos** [comment]
Well,  i'm using Floorp from the Arch repo, (fresh install)  and when i go to bing.com type in a search and hit enter it just enters a space and does nothing can't press enter anymore  without manually clicking back onto the text..   

I've put a video but i think github wouldn't allow it due to size

**Comment by decryptedchaos** [comment]
I was able to compress a video enough to upload. 

https://github.com/user-attachments/assets/4fef9dd0-eaa9-4408-a111-ff460955f64d

**Comment by regularentropy** [comment]
Could you please send the logs from the Floorp DevTools (Ctrl+Shift+Alt+I)?

> I was able to compress a video enough to upload.
> 
>  2024-08-15.2022-14-33.mp4

**Comment by decryptedchaos** [comment]
![image](https://github.com/user-attachments/assets/46304db4-ded2-41dd-8370-02a6fc5ec68d)

the smoking gun is the permission denied error but there's no code in it when you click it it says something like incorrect contents..   

But this doesn't happen on firefox completely different messages in console,  i should also draw attention to the fact that the quick-search thing isn't working (likely due to this JS error)  notice how it isn't showing results as i type..  works in firefox..   

I should also note that i have disabled all ad-blockers etc and its still happening..  this has to be some kind of internal security feature for floorp.

**Comment by surapunoyousei** [comment]
Maybe fixed at 11.17.0
